### PR TITLE
DOC: Review Getting help section

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

File: Documentation/gettinghelp.rst [hunk 1]
```diff
@@ -4,18 +4,23 @@
 Getting Help
 ############
 
-We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions (FAQ)`_.
-Check there to see if your question(s) is already addressed.
+Cilium is a project  with a growing community, so if you need help there a
+numerous ways to getting help:
```
Comment (raybejjani) [inline Documentation/gettinghelp.rst]: "numerous ways to getting it" ? since it repeated help the other way

File: Documentation/gettinghelp.rst [hunk 2]
```diff
@@ -4,18 +4,23 @@
...
 
-The best way to get help if you get stuck is to contact us on the
-`Cilium Slack channel <https://cilium.herokuapp.com>`_.
+**Cilium Frequently Asked Questions (FAQ)**: Cilium uses `GitHub
+tags <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20>`_
```
Comment (raybejjani) [inline Documentation/gettinghelp.rst:11]: Will this parse correctly despite the newline in the middle of the link?

File: Documentation/gettinghelp.rst [hunk 3]
```diff
@@ -4,18 +4,23 @@
...
+to maintain a list of users question. So we suggest you to check if your
+question is already addressed.
```
Comment (raybejjani) [inline Documentation/gettinghelp.rst]: "So we suggest you to check if your question is already addressed." -> "We suggest that you check if your question is already addressed." (I dropped the "So" and add a "that".

File: Documentation/gettinghelp.rst [hunk 4]
```diff
@@ -4,18 +4,23 @@
...
 
-If you are confident that you have found a bug, or if you have a feature
-request, please go ahead and create an issue on our
-`bug tracker <https://github.com/cilium/cilium/issues>`_.
+**Chat**: The best way to get immediate help if you get stuck is to ask in one
+of `Cilium Slack channels <https://cilium.herokuapp.com>`_.
```
Comment (raybejjani) [inline Documentation/gettinghelp.rst]: "one of the"

File: Documentation/gettinghelp.rst [hunk 5]
```diff
@@ -4,18 +4,23 @@
...
+**BugTracker**: All the issues are addressed in `GitHub issue tracker
+<https://github.com/cilium/cilium/issues>`_.  If you want to report a bug or a
+new feature please fill the issue according to the GitHub template.
 
-.. _Cilium Frequently Asked Questions (FAQ): https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20
+**Contributing**: If you want to contribute, reading the :ref:`dev_guide` should
+help you.
+
+**Security**: If you want to report any Security issue please, send an email to
+``security@cilium.io``
```
Comment (raybejjani) [inline Documentation/gettinghelp.rst]: "Security" can be all lowercase, I think?

**Review by raybejjani — APPROVED** [review]
nits nits nits

File: Documentation/gettinghelp.rst (hunk 1 above)
Comment (ianvernon) [reply Documentation/gettinghelp.rst]: I would just change this to "There are numerous ways to get help with Cilium if needed"

File: Documentation/gettinghelp.rst (hunk 3 above)
Comment (ianvernon) [reply Documentation/gettinghelp.rst]: to main a list of questions asked by users. We suggest checking to see if your question is already answered.

File: Documentation/gettinghelp.rst (hunk 5 above)
Comment (ianvernon) [reply Documentation/gettinghelp.rst]: BugTracker --> Bug tracker .

All issues are addressed on the `Github issue tracker...`

fill --> file

File: Documentation/gettinghelp.rst [hunk 6]
```diff
@@ -4,18 +4,23 @@
...
-`Github <https://github.com/cilium/cilium/>`_!
+**BugTracker**: All the issues are addressed in `GitHub issue tracker
+<https://github.com/cilium/cilium/issues>`_.  If you want to report a bug or a
+new feature please fill the issue according to the GitHub template.
 
-.. _Cilium Frequently Asked Questions (FAQ): https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20
+**Contributing**: If you want to contribute, reading the :ref:`dev_guide` should
+help you.
+
+**Security**: If you want to report any Security issue please, send an email to
```
Comment (ianvernon) [inline Documentation/gettinghelp.rst]: If you want to report any security issues within CIlium, please send an email...

**Review by ianvernon — CHANGES_REQUESTED** [review]

File: Documentation/gettinghelp.rst (hunk 2 above)
Comment (eloycoto) [reply Documentation/gettinghelp.rst:11]: Working perfect ;-)

**Comment by eloycoto** [comment]
test-me-please

File: Documentation/gettinghelp.rst [hunk 7]
```diff
@@ -4,18 +4,24 @@
...
 
-.. _Cilium Frequently Asked Questions (FAQ): https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20
+**Bug Tracker**: All the issues are addressed in `GitHub issue tracker
+<https://github.com/cilium/cilium/issues>`_.  If you want to report a bug or a
+new feature please file the issue according to the GitHub template.
+
+**Contributing**: If you want to contribute, reading the :ref:`dev_guide` should
+help you.
+
+**Security**: If you want to report any security issues within CIlium, please
```
Comment (ianvernon) [inline Documentation/gettinghelp.rst]: CIlium --> Cilium

File: Documentation/gettinghelp.rst [hunk 8]
```diff
@@ -4,18 +4,24 @@
...
-`bug tracker <https://github.com/cilium/cilium/issues>`_.
 
-If you are interested in contributing to the code or docs, ping us on
-`Slack <https://cilium.herokuapp.com>`_ or just dive in on
-`Github <https://github.com/cilium/cilium/>`_!
+**Chat**: The best way to get immediate help if you get stuck is to ask in one
+of `Cilium Slack channels <https://cilium.herokuapp.com>`_.
 
-.. _Cilium Frequently Asked Questions (FAQ): https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=label%3Akind%2Fquestion%20
+**Bug Tracker**: All the issues are addressed in `GitHub issue tracker
```
Comment (ianvernon) [inline Documentation/gettinghelp.rst]: ... in the ...

File: Documentation/gettinghelp.rst [hunk 9]
```diff
@@ -4,18 +4,24 @@
...
+<https://github.com/cilium/cilium/issues>`_.  If you want to report a bug or a
+new feature please file the issue according to the GitHub template.
```
Comment (ianvernon) [inline Documentation/gettinghelp.rst]: Is is possible for us to link to this template?

File: Documentation/gettinghelp.rst [hunk 10]
```diff
@@ -4,18 +4,24 @@
...
 
-If you are confident that you have found a bug, or if you have a feature
-request, please go ahead and create an issue on our
-`bug tracker <https://github.com/cilium/cilium/issues>`_.
 
-If you are interested in contributing to the code or docs, ping us on
-`Slack <https://cilium.herokuapp.com>`_ or just dive in on
-`Github <https://github.com/cilium/cilium/>`_!
+**Chat**: The best way to get immediate help if you get stuck is to ask in one
+of `Cilium Slack channels <https://cilium.herokuapp.com>`_.
```
Comment (ianvernon) [inline Documentation/gettinghelp.rst]: ... one of the ...

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-docs-please

**Comment by eloycoto** [comment]
test-docs-please

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**